subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md

```diff
@@ -0,0 +1,23 @@
+#Guidelines for checkout discussion sessions
+
+This document outlines some things that can be discussed during the checkout discussion sessions. The sessions are meant to be student driven, but some groups may be quieter than others. This is by no means an exhaustive list, other subjects are bound to come up. 
+
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
+* Discussion lead creates hangout. Post link to hangout in the session list etherpad.
```

Comment:
I personally prefer hangouts. If we are to use BlueJeans, that means that somebody in SCF needs to do things beforehand. Hangouts can be created by each discussion session leader on their own.